includes/base/controls-stack.php

```diff
@@ -1925,8 +1918,8 @@ protected function init_controls() {
 
 		// TODO: This is for backwards compatibility starting from 2.9.0
 		// This `if` statement should be removed when the method is removed
-		if ( method_exists( $this, '_register_controls' ) ) {
-			$this->_register_controls();
+		if ( method_exists( $this, 'register_controls' ) ) {
```

Comment:
1. We have to keep the old code for now.
2. We should find a way to detect if the method is an inherited method or the parent's one

Reply:
It doesn't work with the old code